crates/sui-core/src/authority/shared_object_congestion_tracker.rs

```diff
@@ -226,14 +226,23 @@ impl SharedObjectCongestionTracker {
             PerObjectCongestionControlMode::TotalGasBudgetWithCap => {
                 Some(std::cmp::min(cert.gas_budget(), self.get_tx_cost_cap(cert)))
             }
-            PerObjectCongestionControlMode::ExecutionTimeEstimate(_) => Some(
-                execution_time_estimator
+            PerObjectCongestionControlMode::ExecutionTimeEstimate(_) => {
+                let estimate_us = execution_time_estimator
                     .expect("`execution_time_estimator` must be set for PerObjectCongestionControlMode::ExecutionTimeEstimate")
                     .get_estimate(cert.transaction_data())
                     .as_micros()
                     .try_into()
-                    .unwrap_or(u64::MAX),
-            ),
+                    .unwrap_or(u64::MAX);
+                if estimate_us >= 15_000 {
+                    let digest = cert.digest();
+                    debug!(
```

Comment:
this feels like it will get very noisy past this immediate crisis, would `trace!` be better or I guess we can always downgrade later?

Reply:
we already have lots or per-tx debug logs, what's one more? especially since this will be only a small subset of txns (we hope). Can always remove later

Reply:
`trace!` would difficult to enable without turning on the firehose